components/04-templates/_default.css

```diff
@@ -4,7 +4,7 @@
 
 $sidebar-width: 33%;
 $gutter: $space;
-$main-width: calc(100% - ($sidebar-width + $gutter));
+$main-width: calc(100% - (#{$sidebar-width} + $gutter));
```

Comment:
this syntax didn't work for me in my original tests. did you test?

Reply:
It did, but it doesn't now. 🤷‍♂ I only added it to avoid the `function-calc-no-invalid` linting error, but since you ignored that in a different PR, I added the ignore line here too. Then we don't have to worry about it.